src/workerd/io/worker-interface.capnp

```diff
@@ -216,6 +216,10 @@ enum SerializationTag {
   headers @4;
   request @5;
   response @6;
+
+  domException @7;
+  # Keep this value in sync with the DOMException::SERIALIZATION_TAG in
```

Comment:
Not that the value can ever change anyway...

Reply:
heh, true. clarified in a comment